src/stars.css

```diff
@@ -0,0 +1,28 @@
+.card-product__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+  line-height: 16px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:nth-child(-n + 4) {
```

Comment:
should also work for other number of stars

Reply:
* not fixed